client/jetpack-connect/auth-logged-out-form.jsx

```diff
@@ -57,21 +57,23 @@ class LoggedOutForm extends Component {
 	}
 
 	renderLoginUser() {
-		const { userData, bearerToken } = this.props.jetpackConnectAuthorize;
-
-		return (
-			<WpcomLoginForm
-				log={ userData.username }
-				authorization={ 'Bearer ' + bearerToken }
-				redirectTo={ this.getRedirectAfterLoginUrl() } />
-		);
+		if ( this.props.jetpackConnectAuthorize ) {
```

Comment:
This seems unrelated to the issue being addressed by this PR, is it possible that this could be falsy?